### PR TITLE
Refactor: added dev guides removed mission guides

### PR DESCRIPTION
**Pull request by rhoads-zach**
@inoxx03 mind taking a look?

@ALRubinger fyi

### CODE REVIEW

File: docs/vertx-runtime/master.adoc [hunk 1]
```diff
@@ -0,0 +1,121 @@
+include::topics/templates/document-attributes.adoc[]
+
+= {vertx-runtime-guide-name}
+
+This guide covers concepts as well as practical details need by developers to use the {VertX} runtime.
```
Comment (inoxx03) [inline docs/vertx-runtime/master.adoc]: Shouldn't it be _needed_ instead of _need_?

File: docs/vertx-runtime/master.adoc [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+include::topics/secured-mission-deploy-rhsso.adoc[leveloffset=+2]
+
+include::topics/secured-mission-deploy-booster.adoc[leveloffset=+2]
+
+include::topics/secured-mission-common-interaction.adoc[leveloffset=+2]
+
+
+
+== Creating a Application from Scratch
```
Comment (inoxx03) [inline docs/vertx-runtime/master.adoc]: Creating _an_ Application

File: docs/vertx-runtime/master.adoc [hunk 3]
```diff
@@ -0,0 +1,121 @@
+include::topics/templates/document-attributes.adoc[]
+
+= {vertx-runtime-guide-name}
+
+This guide covers concepts as well as practical details need by developers to use the {VertX} runtime.
+
+== Runtime details
```
Comment (inoxx03) [inline docs/vertx-runtime/master.adoc]: this heading should be in title case.

File: docs/wf-swarm-runtime/master.adoc [hunk 4]
```diff
@@ -0,0 +1,125 @@
+include::topics/templates/document-attributes.adoc[]
+
+= {wf-swarm-runtime-guide-name}
+
+This guide covers concepts as well as practical details need by developers to use the {WildFlySwarm} runtime.
```
Comment (inoxx03) [inline docs/wf-swarm-runtime/master.adoc]: please see above

File: docs/wf-swarm-runtime/master.adoc [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+include::topics/secured-mission-deploy-rhsso.adoc[leveloffset=+2]
+
+include::topics/secured-mission-deploy-booster.adoc[leveloffset=+2]
+
+include::topics/secured-mission-common-interaction.adoc[leveloffset=+2]
+
+
+
+
+== Creating a Application from Scratch
```
Comment (inoxx03) [inline docs/wf-swarm-runtime/master.adoc]: please see above

File: docs/spring-boot-runtime/master.adoc [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+
+include::topics/secured-mission-springboot-adapter.adoc[leveloffset=+2]
+
+include::topics/secured-mission-deploy-rhsso.adoc[leveloffset=+2]
+
+include::topics/secured-mission-deploy-booster.adoc[leveloffset=+2]
+
+include::topics/secured-mission-common-interaction.adoc[leveloffset=+2]
+
+== Creating a Application from Scratch
```
Comment (inoxx03) [inline docs/spring-boot-runtime/master.adoc]: this should probably be
> Creating _an_ Application

File: docs/spring-boot-runtime/master.adoc [hunk 7]
```diff
@@ -0,0 +1,121 @@
+include::topics/templates/document-attributes.adoc[]
+
+= {spring-boot-runtime-guide-name}
+
+This guide covers concepts as well as practical details need by developers to use the {SpringBoot} runtime.
+
+
+== Runtime details
```
Comment (inoxx03) [inline docs/spring-boot-runtime/master.adoc]: This should probably be in title case.

File: docs/spring-boot-runtime/master.adoc [hunk 8]
```diff
@@ -0,0 +1,121 @@
+include::topics/templates/document-attributes.adoc[]
+
+= {spring-boot-runtime-guide-name}
+
+This guide covers concepts as well as practical details need by developers to use the {SpringBoot} runtime.
```
Comment (inoxx03) [inline docs/spring-boot-runtime/master.adoc]: need > needed ?

**Comment by rhoads-zach** [comment]
merged: https://github.com/openshiftio/appdev-documentation/commit/cc61880c8c7296d77a0e5eeb36184407b9e65df2

**Pull request closed without merging**